en/gps_compass/README.md

```diff
@@ -46,8 +46,12 @@ Device | GPS | Compass | [RTK](../gps_compass/rtk_gps.md) | [GPS Yaw Output](#co
 [Holybro Nano Ublox M8 5883 GPS](https://holybro.com/products/nano-m8-5883-gps-module) | UBX-M8030 | QMC5883 | | | |
 [Holybro M8N GPS](../gps_compass/gps_holybro_m8n_m9n.md) | M8N | IST8310 | | | |
 [Holybro M9N GPS](../gps_compass/gps_holybro_m8n_m9n.md) | M9N | IST8310 | | | |
+[Holybro DroneCAN M8N GPS](https://holybro.com/products/dronecan-m8n-gps) | M8N | BMM150 | | | |
```

Comment:
Better to have PX4 docs pages and link to those. But either way, this page already exists above linked to our page.
```suggestion
```